clippy_lints/src/methods/mod.rs

```diff
@@ -1010,6 +1010,12 @@ declare_clippy_lint! {
     "`.chcked_add/sub(x).unwrap_or(MAX/MIN)`"
 }
 
+declare_clippy_lint! {
+    pub FILETYPE_IS_FILE,
+    style,
+    "`FileType.is_file` does not cover all files the can be read."
```

Comment:
```suggestion
    "`FileType::is_file` does not cover all files that can be read"
```
Also exchange the `.` with `::` below.